caffe2/python/serialized_test/serialized_test_data.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from collections import namedtuple
+
+SerializedTestData = namedtuple('SerializedTestData', [
```

Comment:
I guess we probably don't need this, since we can use the folder to enocde test type and anme.
Operators can be serilized as op.pb, and you probably want to support multiple sets of inputs and outputs.